doc/classes/Line2D.xml

```diff
@@ -70,6 +70,13 @@
 			[b]Note:[/b] The shape of the closing segment is not guaranteed to be seamless if a [member width_curve] is provided.
 			[b]Note:[/b] The joint between the closing segment and the first segment is drawn first and it samples the [member gradient] and the [member width_curve] at the beginning. This is an implementation detail that might change in a future version.
 		</member>
+		<member name="curve_line_segments" type="int" setter="set_curve_line_segments" getter="get_curve_line_segments" default="1">
```

Comment:
I'd rename this property to `curve_min_line_segments`, since it's a minimum value.